VPInteractiveImageViewController/VPInteractiveImageViewController/VPInteractiveImageViewController.m

```diff
@@ -30,6 +32,17 @@ - (instancetype)initWithInteractiveImageView:(VPInteractiveImageView *)interacti
     return self;
 }
 
+- (instancetype)initWithNibName:(nullable NSString *)nibNameOrNil
```

Comment:
Even with default initializers? That would be the nicer solution. I wil try it out, thanks!